lib/eos/ecc/point.js

```diff
@@ -1,10 +1,24 @@
-var assert = require('assert')
-var BigInteger = require('bigi/lib/bigi')
+import bigInt from 'big-integer';
+import { assert } from './../helper';
 
-var THREE = BigInteger.valueOf(3)
+const ONE = bigInt.one;
+const ZERO = bigInt.zero;
+const THREE = bigInt(3);
 
-function Point (curve, x, y, z) {
-  assert.notStrictEqual(z, undefined, 'Missing Z coordinate')
+function testBit(bigIntVal, index) {
+  return bigIntVal.toString(2)[index] === '1';
+}
+
+function bufferToBigInt(buffer) {
+  return bigInt(buffer.toString('hex'), 16);
```

Comment:
Does `bigInt` not have a constructor that accepts a buffer?

Reply:
Unfortunately `bigInt` doesn't have a built-in constructor that accepts a buffer

Reply:
I think `toArray` and `fromArray` with radix 256 should work here.

Reply:
Fixed